config/config.go

```diff
@@ -199,22 +201,47 @@ func get(args []string, l *logrus.Entry) (cfg *Config, configFileF string, err e
 			&cfg.Paths.AzureExporter:    "azure_exporter",
 			&cfg.Paths.VMAgent:          "vmagent",
 			&cfg.Paths.TempDir:          os.TempDir(),
-			&cfg.Paths.PTSummary:        "/usr/local/percona/pmm2/tools/pt-summary",
-			&cfg.Paths.PTPgSummary:      "/usr/local/percona/pmm2/tools/pt-pg-summary",
-			&cfg.Paths.PTMongoDBSummary: "/usr/local/percona/pmm2/tools/pt-mongodb-summary",
-			&cfg.Paths.PTMySqlSummary:   "/usr/local/percona/pmm2/tools/pt-mysql-summary",
+			&cfg.Paths.PTSummary:        "tools/pt-summary",
+			&cfg.Paths.PTPgSummary:      "tools/pt-pg-summary",
+			&cfg.Paths.PTMongoDBSummary: "tools/pt-mongodb-summary",
+			&cfg.Paths.PTMySqlSummary:   "tools/pt-mysql-summary",
 		} {
 			if *sp == "" {
 				*sp = v
 			}
 		}
 
+		if cfg.Paths.PathsBase == "" {
+			cfg.Paths.PathsBase = pathBaseDefault
+		}
+		if cfg.Paths.ExportersBase == "" {
+			cfg.Paths.ExportersBase = filepath.Join(cfg.Paths.PathsBase, "exporters")
+		}
+
+		if cfg.Paths.PathsBase != "" {
```

Comment:
this condition seems to be uncessary. because `pathsBase` can't be never empty on this line.
The same below with exporters base.